modules/040-node-manager/openapi/values.yaml

```diff
@@ -338,6 +339,15 @@ properties:
               # This is a copy from NodeGroup object. We trust that hook will not change it.
               type: object
               additionalProperties: true
+            fencing:
+              # This is a copy from NodeGroup object. We trust that hook will not change it.
+              type: object
+              properties:
+                mode:
+                  type: string
+                  enum:
+                    - "Disabled"
```

Comment:
Usually we do not use negative enum values. It is better to use `Enabled` instead of `Disabled`. In your case, this is more problematic, because not specifying the fencing block and specifying `fancing: {mode: Disabled}` is actually the same thing. 

Let's use enum with a single value `Watchdog` for now. If there is no fencing block, it will mean that the fencing feature is disabled.

Reply:
done